exdb/templates/exdb/ra_home.html

```diff
@@ -0,0 +1,32 @@
+{% extends "exdb/base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="{% static 'exdb/css/ra-home.css' %}"/>
+{% endblock %}
+{% block content %}
+    {{ block.super }}
+    <h2>{% trans 'Hello' %} {{ ra }}</h2>
+    <hr /><hr />
+    {% for status, experiences in experience_dict.items %}
+        {% if experiences|length %}
+            <h1>{{ status }}</h1>
+            <table class="experience-list">
```

Comment:
I don't know that this data is best represented in a table. We should discuss this at some point.